app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Api::V1::ItemsController < ApplicationController 
+
+  def index
+    if params[:merchant_id]
+      render json: ItemSerializer.new(Merchant.find(params[:merchant_id]).items)
+    else
+      render json: ItemSerializer.new(Item.all)
+    end
```

Comment:
returns the `merchant` `items` if `merchant_id` is passed in the params or all the `items` otherwise